README.md

```diff
@@ -1,19 +1,19 @@
 <div aria-hidden="true">
 
 [![Latest Stable Version](https://poser.pugx.org/wp-coding-standards/wpcs/v/stable)](https://packagist.org/packages/wp-coding-standards/wpcs)
-[![Travis Build Status](https://travis-ci.org/WordPress-Coding-Standards/WordPress-Coding-Standards.svg?branch=master)](https://travis-ci.org/WordPress-Coding-Standards/WordPress-Coding-Standards)
-[![Release Date of the Latest Version](https://img.shields.io/github/release-date/WordPress-Coding-Standards/WordPress-Coding-Standards.svg?maxAge=1800)](https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/releases)
+[![Travis Build Status](https://api.travis-ci.org/WordPress/WordPress-Coding-Standards.svg?branch=master)](https://travis-ci.org/WordPress/WordPress-Coding-Standards)
+[![Release Date of the Latest Version](https://img.shields.io/github/release-date/WordPress/WordPress-Coding-Standards.svg?maxAge=1800)](https://github.com/WordPress/WordPress-Coding-Standards/releases)
 :construction:
 [![Latest Unstable Version](https://img.shields.io/badge/unstable-dev--develop-e68718.svg?maxAge=2419200)](https://packagist.org/packages/wp-coding-standards/wpcs#dev-develop)
-[![Travis Build Status](https://travis-ci.org/WordPress-Coding-Standards/WordPress-Coding-Standards.svg?branch=develop)](https://travis-ci.org/WordPress-Coding-Standards/WordPress-Coding-Standards)
-[![Last Commit to Unstable](https://img.shields.io/github/last-commit/WordPress-Coding-Standards/WordPress-Coding-Standards/develop.svg)](https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/commits/develop)
+[![Travis Build Status](https://travis-ci.org/WordPress/WordPress-Coding-Standards.svg?branch=develop)](https://travis-ci.org/WordPress/WordPress-Coding-Standards)
```

Comment:
Does this need the `api.` subdomain like the previous example?

Reply:
Yeah, since travis redirects me to `api.` url I should change this as well, good catch :+1:

Reply:
I'm sorry to be a pain, but I don't think this is right.

When I look at what Travis provides as code to get the images, it doesn't include the `api.` bit, so that should be regarded as an internal redirect and the link they provide via the image in the build screen as the canonical.

![WPCS Travis links](https://user-images.githubusercontent.com/663378/60218542-ee668a00-986f-11e9-8a96-50ecf9537a31.png)

Reply:
Yeah, when I entered the link it redirected me so I thought that's the proper one. I'll remove the `api.` prefix 🙂